src/components/toast/index.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { useToast } from 'src/hooks';
+
+import { Button, ErrorIcon } from '../index.ts';
+
+<Meta title="Components/Feedback/Toast" component={useToast} />
+
+export const Template = (args) => {
+  const toast = useToast();
+  return (
+    <Button
+      onClick={() =>
+        toast({
+          type: args.type,
+          title: 'Your browser is outdated!',
+          description: 'Your Chakra experience may be degraded.',
+          icon: <ErrorIcon />,
+        })
+      }
+    >
+      Show Toast
+    </Button>
+  );
+};
+
+# Toast
+
+<Canvas>
+  <Story
+    name="Overview"
```

Comment:
Can you name it as the componen? We want to avoid to add more tree in the categories